client-src/src/app/app.component.ts

```diff
@@ -7,7 +7,7 @@ import { InqProvider } from '../providers/inq/inq';
 import { FollowUpProvider } from '../providers/follow-up/follow-up';
 
 import { HomePage } from '../pages/home/home';
-import { CounselorHomePage } from '../pages/counselor/home/home';
+import { CounselorDashboardPage } from '../pages/counselor/home/home';
```

Comment:
Create a low priority defect to rename all and remove page from the view name. Use CounsellorHome or CounsellorHomeView for example

Reply:
Okay